.github/workflows/build-test.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
       - run: npm ci
+      - name: Install Playwright
+        run: npx playwright install chromium
```

Comment:
This should probably be moved to our `prepare` script, all contributors will need this ran too.